src/components/Block.js

```diff
@@ -74,6 +79,7 @@ export default class Block extends React.PureComponent {
           <Component
             ref={el => (this.block = el)}
             {...block}
+            {...options}
```

Comment:
I'm worried this will cause bugs where an option field could clear important block properties. What if this were passed as `options={options}`?